distributed/pom.xml

```diff
@@ -135,6 +135,11 @@
             <artifactId>hazelcast</artifactId>
             <version>${hz.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Is this a compile time dependency or a runtime dependency? HZ hast a tons of different plugins, i would prefer to avoid to add all of them as dependency if is not needed, but yes not having this as the dependency may add additional steps to the deploy procedure on kubernates, not strongly against this, but if is not needed i would prefer to keep the dependency out.